src/ParallelAlgorithms/Actions/FunctionsOfTimeAreReady.hpp

```diff
@@ -143,6 +143,32 @@ bool functions_of_time_are_ready_simple_action_callback(
       std::forward<Args>(args)...);
 }
 
+/// \ingroup ComputationalDomainGroup
+/// Check that functions of time are up-to-date.
+///
+/// Check that functions of time in \p CacheTag with names in \p
+/// functions_to_check are ready at time \p time.  If  \p functions_to_check is
+/// a `std::nullopt`, checks all functions in \p CacheTag.  If any function is
+/// not ready, schedules a `Parallel::ThreadedActionCallback` with the
+/// GlobalCache which calls the threaded action passed in as a template
+/// parameter. The `Args` are moved into the callback.
```

Comment:
They're forwarded, not moved. There's a difference, right?

Reply:
Yep, I'll also update the simple action variant to say forwarded as well!